macosx/PeerProgressIndicatorCell.m

```diff
@@ -53,8 +53,8 @@ - (void) drawWithFrame: (NSRect) cellFrame inView: (NSView *) controlView
             NSMutableParagraphStyle * paragraphStyle = [[NSParagraphStyle defaultParagraphStyle] mutableCopy];
             [paragraphStyle setAlignment: NSRightTextAlignment];
 
-            fAttributes = [[NSDictionary alloc] initWithObjectsAndKeys: [NSFont systemFontOfSize: 11.0], NSFontAttributeName,
-                                                                            paragraphStyle, NSParagraphStyleAttributeName, nil];
+            fAttributes = @{NSFontAttributeName: [NSFont systemFontOfSize: 11.0],
```

Comment:
Wasn't autoreleased before, explicit release happens in `dealloc` and 12 lines down